### PR TITLE
A webpage cloning newyork times webpage 

### PR DESCRIPTION
**Pull request by danmainah**
this project consists of making a clone of a New York Times article page.
It has different sections . i have implemented the various sections :
navbar
main
sidebar
footer
iframe

### CODE REVIEW

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Invalid Code Review Request :x: 

__Hi @danmainah, I commend you for the great effort put into this project. However, I have to mark the project as invalid due to the following reasons:__

- [ ] Your linter checks are not running: to consider this project ready for review, you need to ensure that all GitHub Action Checks are configured and running successfully.
- [ ] As a correction, you submitted the wrong link on the review request form. For subsequent submissions, use the PR page link (i.e. https://github.com/danmainah/week1-project/pull/1)
- [ ] You should ensure that your PR has a descriptive title and summary. 
- [ ] You should also present a descriptive Readme for your project. [This template](https://github.com/microverseinc/readme-template) might help.


Friendly reminder: You should make sure that your repo looks professional before review.
Please take a step back and read a lesson [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/).
Do not forget to apply 3 simple rules from this lesson (it will be easy as you can use a template provided there). This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!

* * *
**Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * * 

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Comment by danmainah** [comment]
Thank you @teekaytech for the feedback will check on them.

**Review by teekaytech — CHANGES_REQUESTED** [review]
### Status: Invalid Code Review Request :x: 

__Hi @danmainah :wave: I commend you for the great effort put into this project. However, I have to mark the project as invalid due to the following reasons:__

- [ ] Although you've configured the GitHub Action checks, it seems not to be running. You need to ensure that all GitHub Action Checks are running successfully. 
- [ ] The PR summary is still missing. 

Do you need help? You can register for one-on-one session with a full-time TSE [here](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F360026625973-How-can-I-get-help-from-TSEs-).
This Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready!


* * *
**Please, do not open a new Pull Request for subsequent reviews. You should use the same Pull Request submitted for the first review unless it is requested otherwise.**

**Always post your inquiries and questions on this PR thread as a comment. You can mention me (@teekaytech) so I can see your comments and respond.**

* * * 

**_You can reach out to me on Twitter (@ola_lere), Github(@teekaytech), or slack (@taofeekolalere )_**

Happy Coding :man_technologist:

**Review by voscarmv — CHANGES_REQUESTED** [review]
Hi @danmainah,

As the previous Code Reviewer mentioned, your GitHub Actions tests are failing. Mainly, you have a series of CSS linter errors:

image

Because of this, I will have to invalidate your project yet once again. But don't worry:

You can use the `npx stylelint "**/*.{css,scss}" --fix` command in your local environment to automatically fix these errors before submitting again. Please read the instructions here to learn more about how to do this: https://github.com/microverseinc/linters-config/tree/master/html-css

If you need any help feel free to contact me on Slack, my handle is Oscar Mier.
Cheers and Happy Coding!

**Review by Temmarie — CHANGES_REQUESTED** [review]
## Changes Required! :recycle:

### Hi @danmainah   ! :wave: :wave:

#### You have done well implementing the previously required changes :clap: and good job using semantic HTML tags 👍 .  However, there are changes required for you to make in order for this project to be approved. **_Note that you have two more reviews on this project before you reach the review limit, so make all of the required changes before submitting for a review._**
- - -

### **Recommended**

**General**
- [ ] Remove the comments copied from the template file you used for your readme and follow the steps suggested there instead of just copying the template.

**Section Specific requirements**
 - [ ] Some parts of the main content texts have fonts that are larger than what is on the page. I'll recommend using [this](https://chrome.google.com/webstore/detail/font-picker/mmjbimgpcbaegjiieojddickpjbdkeej) chrome extension tool to help with this.
 - [ ] Most of the links on the page are still retaining the styles that come with the `a` tag. Kindly remove those link styles. 
 - [ ] Fix your navbar, it has some issues.
 - [ ] The main image of Alan Guth is a lot bigger than that on the actual site.
 - [ ] The Theory of Inflation section needs to be fixed and aligned to the center. Part of the issue here is the size of the font.
 - [ ] The following sections need to be fixed:
 - The Suggested Newsletter,
 - The Most Popular,
 - More In Space and Astronomy,
 - The Editors' Picks and
 - The footer.
 The images and texts are not positioned as they should be. Please fix them and remember to remove the link styles from the parts of the project necessary.
In conclusion, kindly refer to the actual site and compare your page to it and see the necessary changes you have to make.

__Kindly make the necessary changes and submit another code review request.__
_Feel free to reach me on slack for any inquiries about this review._ :wink:
__Grace Tamara Ekunola __
[GitHub](https://github.com/temmarie) | [LinkedIn](https://www.linkedin.com/in/ekunola-grace/) | [Twitter](https://twitter.com/TemmarieW)

**Review by descholar-ceo — CHANGES_REQUESTED** [review]
# Status: Changes Required :vertical_traffic_light: :warning: 
## Hello @danmainah  :handshake: :handshake: :handshake: 

:+1: This is good to make it this far, your clone almost looks like the original page and it is really nice, congratulations on the good work you have done so far.
:+1: I liked that you have tried to implemented the requested changes from the previous review, keep it up, and your project is really almost done.
#### :warning: However we need to make few changes to mark this project approved, don't feel discouraged because of being asked some changes more and more, that's because this is a learning process, and we are here to help you to become an out-standing developer. See the requested change below :point_down: :point_down: :point_down: 
### Note: All of requested changes which are marked as [Required] are mandatory to be implemented in order to get this project approved, others which are marked as [Optional] are nice to have in your project to make it outstanding

# [Required Changes] :red_circle: 
- [ ] The navbar still needs more work, please make sure it looks like the original page
- [ ] The comments button on the bottom part of the main content needs to be fixed and aligned
- [ ] You need to remove the `.idea` folder from the root directory of your project, it is not related to this project
- [ ] The following sections need to be fixed:
    - [ ] The Suggested Newsletter,
    - [ ] The Most Popular,
    - [ ] More In Space and Astronomy,
    - [ ] The Editors' Picks and

:point_right: The images and texts are not positioned as they should be. Please fix them, that section should be aligned in the middle and not taking the whole width of the viewport.
In fact, kindly refer to the actual site and compare your page to it and see the necessary changes you have to make.
___

### :warning: Warning: According to the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561#) you have 1 code review left for this project, make sure that you implement all of the required changes carefully and request another review while you are sure that it's ready. :warning: 

_If anything seems to be unclear to you, reachout and ask anything by commenting on this Pull Request and mention me as [descholar-ceo](https://github.com/descholar-ceo), I will receive a notification and will respond._

[GitHub](https://github.com/descholar-ceo) | [LinkedIn](https://linkedin.com/in/mugirase-emmanuel) | [Twitter](https://twitter.com/descholar3)

_Reviewed by [Mugirase Emmanuel (descholar)](https://github.com/descholar-ceo)_
